projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.ts

```diff
@@ -155,6 +155,14 @@ export class MultiLayerSelectionComponent implements ControlValueAccessor, OnCha
         }
     }
 
+    updateLayer(index: number, layer: Layer): void {
+        this.selectedLayers.pipe(first()).subscribe((selectedLayers) => {
```

Comment:
I think you can directly access the selected layer using `this.selectedLayers.value` instead of subscribing here.

Reply:
That's actually much shorter: ad2881d1a5fd39c1fb4937f22dfd8f60978e76ba